### PR TITLE
[TTS] HiFi-TTS Download Script

### PR DESCRIPTION
**Pull request by oleksiivolk**
# What does this PR do ?

Download hifitts dataset and reformat into ljspeech format.

**Collection**: [Note which collection this PR will affect]

# Usage

```python
python scripts/dataset_processing/tts/hifitts/get_data.py --data-root /home/ovolkovskyi/NeMo/
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

**Comment by XuesongYang** [comment]
> This pull request **introduces 3 alerts** when merging [70d02c1](https://github.com/NVIDIA/NeMo/commit/70d02c11f7f844f2cbc96fd65ede03782152aa99) into [ea02fb2](https://github.com/NVIDIA/NeMo/commit/ea02fb2b3d4be65007b8f3f4bc0cbdfdccf5e249) - [view on LGTM.com](https://lgtm.com/projects/g/NVIDIA/NeMo/rev/pr-1bb5932d97ba8ba4582cc2a9b6a173816d6776ce)
> 
> **new alerts:**
> 
> * 3 for Unused import

please remove these ^ and signoff. Thanks.

File: scripts/dataset_processing/tts/hifitts/get_data.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import glob
+import json
+import re
+import tarfile
+import urllib.request
+from email.mime import audio
```
Comment (subhankar-ghosh) [inline scripts/dataset_processing/tts/hifitts/get_data.py:21]: Unused import

File: scripts/dataset_processing/tts/hifitts/get_data.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+from pathlib import Path
+
+import sox
```
Comment (subhankar-ghosh) [inline scripts/dataset_processing/tts/hifitts/get_data.py:24]: Unused import

File: scripts/dataset_processing/tts/hifitts/get_data.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+import wget
```
Comment (subhankar-ghosh) [inline scripts/dataset_processing/tts/hifitts/get_data.py:25]: unused import

File: scripts/dataset_processing/tts/hifitts/get_data.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+from tqdm import tqdm
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Download HiFiTTS and create manifests with predefined split')
+    parser.add_argument("--data-root", required=True, type=Path)
```
Comment (subhankar-ghosh) [inline scripts/dataset_processing/tts/hifitts/get_data.py:31]: Add description for each argument.

File: scripts/dataset_processing/tts/hifitts/get_data.py [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+                    wav_path = data_root / data['audio_filepath']
+                    assert wav_path.exists(), f"{wav_path} does not exist!"
+
+                    entry = {
+                        'audio_filepath': data['audio_filepath'],
+                        'duration': data['duration'],
+                        'text': data['text'],
+                        'normalized_text': data['text_normalized'],
+                        'speaker': int(speaker_id),
+                        'audio_quality': audio_quality,
```
Comment (subhankar-ghosh) [inline scripts/dataset_processing/tts/hifitts/get_data.py:87]: What are the values of `audio_quality`?

**Review by subhankar-ghosh — CHANGES_REQUESTED** [review]
Left few comments.

File: scripts/dataset_processing/tts/hifitts/get_data.py (hunk 5 above)
Comment (oleksiivolk) [reply scripts/dataset_processing/tts/hifitts/get_data.py:87]: Audio_quality is either "clean" or "other". 
The clean set includes recordings with high sound-to-noise ratio and wide bandwidth.
The books with noticeable noise or narrow bandwidth are included in the other subset.
Note: some speaker_id's have both clean and other audio quality.

Comment (oleksiivolk) [reply scripts/dataset_processing/tts/hifitts/get_data.py:87]: Added this excerpt as a comment in the code too.

**Review by subhankar-ghosh — APPROVED** [review]
LGTM

**Pull request merged**